.gitignore

```diff
@@ -57,8 +57,8 @@ vendor/bundle
 .ignore
 TODO
 
-
 node_modules
 app/assets/javascripts/spree-dashboard.js
 app/assets/javascripts/spree-dashboard.*.js
 src
+yarn.lock
```

Comment:
@MatthewKennedy when we ignore yarn.lock when building the package it will fetch the newest versions of packages (at least the newest ones allowed by the version requirement string) which can create some side effects

Reply:
@damianlegawiec Can you not lock the versions in the package.json like you would in a gem file.

Reply:
@MatthewKennedy we could but later tools like dependabot wouldn't create PRs with upgrades ;)